src/modules/esl-footnotes/core/esl-note.ts

```diff
@@ -7,7 +7,7 @@ import {ESLTooltip} from '../../esl-tooltip/core';
 import {promisifyTimeout, repeatSequence} from '../../esl-utils/async/promise';
 import {EventUtils} from '../../esl-utils/dom/events';
 import {ENTER, SPACE} from '../../esl-utils/dom/keys';
-import {scrollIntoView} from '../../esl-utils/dom/scroll';
+import {scrollIntoView} from '../../esl-utils/misc/scrollIntoView';
```

Comment:
The module is incorrect `esl-utils/dom/scroll` should be used instead (module name does not depends on the fact of code splitting, see `dom/events` as the reference of structural approach)